src/ray/stats/stats.h

```diff
@@ -50,14 +50,23 @@ static inline void Init(
     RAY_LOG(INFO) << "Disabled stats.";
     return;
   }
+  RAY_LOG(INFO) << "Initialize stats";
```

Comment:
Can we make these logs DEBUG level? This line will be included in driver logs, right? I don't think users should care about this.

Same for other logs in this file unless they are warnings or errors.

Reply:
> Can we make these logs DEBUG level? This line will be included in driver logs, right? I don't think users should care about this.
> 
> Same for other logs in this file unless they are warnings or errors.

Nice catch. I'll remove it later.